version/version.go

```diff
@@ -1,4 +1,51 @@
 package version
 
+import (
+	"strings"
+
+	"github.com/blang/semver"
+	"github.com/google/renameio"
+)
+
 // Version is the version of the build.
 const Version = "1.14.2-dev"
+
+// WriteVersionFile writes the version information to a given file
+// file is the location of the old version file
+// gitCommit is the current git commit version. It will be added to the file
+// to aid in debugging, but will not be used to compare versions
+func WriteVersionFile(file string, gitCommit string) error {
```

Comment:
Some unit tests would be great to see here. :innocent:

Reply:
Added :)